spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,39 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  describe "I am taken to '/quotes' where I see the TOTAL number of quotes that were found for that keyword" do
```

Comment:
Remember back to Mod 2 and using within blocks? You will want to use those in this test since we should be returning 10 results. So you could do something like
```
within(first(".quotes")) do
    expect(page).to have_content("Author: Rabbi Hillel")
    expect(page).to have_content("Quote: If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?")
    expect(page).to have_content("Category(ies): famous-quotes, wisdom")
end